lisa/tools/iperf3.py

```diff
@@ -466,6 +463,6 @@ def _get_bandwidth(self, result: str, pattern: Pattern[str]) -> Decimal:
 
     def _pre_handle(self, result: str) -> str:
         result = result.replace("-nan", "0")
-        result_matched = self._json_pattern.match(result)
+        result_matched = self._json_pattern.search(result)
```

Comment:
Try if it's possible to use lisa internal wrappers.

Reply:
update